src/modules/service-catalog/components/service-catalog-item/CollapsibleDescription.tsx

```diff
@@ -59,8 +67,7 @@ export const CollapsibleDescription = ({
   const [isExpanded, setIsExpanded] = useState<boolean>(false);
   const { t } = useTranslation();
 
-  const showToggleButton = description.length > DESCRIPTION_LENGTH_THRESHOLD;
-
+  const showToggleButton = shouldShowToggleButton(description);
```

Comment:
We don't need additional function, something like this should be enough: 
`const showToggleButton = description != null && description.length > DESCRIPTION_LENGTH_THRESHOLD;`

Reply:
I extracted it so I could test it, otherwise it can't be tested in isolation

Reply:
We don't have tests in this repo, we usually write e2e tests for CPH theme in Theming Center

Reply:
its not a dependency install, no new packages were added - just a mock, and this is the same we do for testing in other jest cases. As soon as we increase our test coverage - which I don't understand quite why we don't have more tests on this repo already - it becomes needed, so unsure why not add it now?